website/pages/docs/secrets/kv/index.mdx

```diff
@@ -13,6 +13,8 @@ modes. It can be a generic Key-Value store that stores one value for a key.
 Versioning can be enabled and a configurable number of versions for each key
 will be stored.
 
+Note that `kv` secrets mounts are case-sensitive.
```

Comment:
What does this mean?  That we preserve case for keys?

Reply:
> What does this mean? That we preserve case for keys?

@ncabatoff  apparently so?  https://app.asana.com/0/112957393038545/1109620483178716/f